composer.json

```diff
@@ -38,19 +38,26 @@
 		"docraptor/docraptor": "^3.0",
 		"fale/isbn": "^3.0",
 		"gridonic/princexml-php": "^1.2",
-		"illuminate/container": "5.4.*",
-		"jenssegers/blade": "^1.1",
+		"illuminate/container": "^8.0",
+		"illuminate/contracts": "^8.0",
+		"illuminate/events": "^8.0",
+		"illuminate/http": "^8.0",
+		"illuminate/filesystem": "^8.0",
+		"illuminate/support": "^8.0",
+		"illuminate/view": "^8.0",
 		"jenssegers/imagehash": "0.5.0",
 		"johnbillion/extended-cpts": "^5.0",
+		"laravel/helpers": "^1.5",
 		"masterminds/html5": "^2.6",
 		"maxbanton/cwh": "^2.0",
 		"monolog/monolog": "^2.2",
 		"pressbooks/mix": "^2.1",
-		"pressbooks/pb-cli": "^2.1.0",
+		"pressbooks/pb-cli": "dev-dev",
```

Comment:
Shouldn't we require a tagged version of this repo? If you need something that's in dev branch, we can always cut a new release ...